targets/ramips-mt7621

```diff
@@ -8,7 +8,8 @@ device d-link-dir-860l-b1 dir-860l-b1
 
 ## ZBT
 
-device zbt-wg3526 zbt-wg3526
+device zbt-wg3526-16M zbt-wg3526-16M
+device zbt-wg3526-32M zbt-wg3526-32M
 factory
```

Comment:
there is no factory - there is only a sysupgrade on build (https://github.com/FreifunkBremen/gluon-site-ffhb/commit/c8dec17d7c75e7649c6753f3405abeef8a37f5a2):
```
ls gluon/lede/bin/targets/ramips/mt762:
lede-ramips-mt7621-zbt-wg3526-32M-squashfs-sysupgrade.bin
```

Reply:
@CodeFetch better at this line?

Reply:
How is that review helpful?

The correct lines would be:
```
-device zbt-wg3526 zbt-wg3526
+device zbt-wg3526-16M zbt-wg3526-16M
+factory
+device zbt-wg3526-32M zbt-wg3526-32M
```

The blank factory line says "There is no factory-image".

Reply:
it's correct how it is in the current PR status.
"factory" means, that there is NO factory. a bit misleading, but it is like it is.